vllm/entrypoints/llm.py

```diff
@@ -202,11 +207,12 @@ def _run_engine(self, use_tqdm: bool, profiling: bool = False) -> List[RequestOu
                     outputs.append(output)
                     if use_tqdm:
                         pbar.update(1)
-            if profiling and is_hpu():
-                htorch.core.mark_step()
+            if profiling:
                 prof.step()
-        if profiling and is_hpu():
-            htorch.hpu.synchronize()
+        if profiling:
+            if is_hpu():
+                import habana_frameworks.torch as htorch
```

Comment:
Is there a reason to place this import here instead of at the top of the file?

Reply:
That was the only place where we needed that import. Anyway, I'll move it to the top (with a conditional on is_hpu) so that it's consistent with entrypoints/api_server.py .

Reply:
Done. Please let me know if it looks ok :)